src/transformers/quantizers/quantizer_bnb_4bit.py

```diff
@@ -121,7 +121,7 @@ def check_quantized_param(
         import bitsandbytes as bnb
 
         module, tensor_name = get_module_from_name(model, param_name)
-        if isinstance(module._parameters[tensor_name], bnb.nn.Params4bit):
+        if tensor_name in module._parameters and isinstance(module._parameters[tensor_name], bnb.nn.Params4bit):
```

Comment:
```suggestion
        if isinstance(module._parameters.get(tensor_name,None), bnb.nn.Params4bit):
```
a nit but it's equivalent no?